### DISCUSSION TITLE
rotate:45deg without transform:

### DISCUSSION THREAD

**Discussion by rafehimself** [cssbattle/forum#169] [Q&A] [answered]
I saw on youtube Kevin rotate square just with that` rotate:45deg`
https://youtu.be/Rf8XDHCMogM?t=591
time 9:35
```
<div></div>
<style>
  body{display:flex;
  justify-content:center;
  place-items:center;}
  div {
    width: 100px;
    height: 100px;
    background: #dd6b4d;
    rotate: 45deg;
  }
</style>
```
but when I try to do it, it doesn't work 🤣 c'mon that's free 9 chars, I would take it

**Comment by chinchang** [comment, marked as answer]
@rafehimself It's not yet supported on Chromium stable. Just Firefox works for now.

**Reply by rafehimself** [reply]
I hope that will be implemented soon :), that transform is not necessary

**Comment by Beowolve** [comment]
You can use `offset:path` as a replacement for `transform:rotate` in many places. It's a very nice trick I discovered last battle from [target#84](https://cssbattle.dev/play/84). The code actually isn't shorter just for rotation, but you can also offset your element and save chars in total.

```
<style>*{background:#F3AC3C}
*>*{
  margin:135 185;
  background:#1A4341;
  offset:path('M13 15l1 1')
}
```

**Reply by GaganPrasad** [reply]
Wow thanks made it 100%, was trying was not able to make it 100% with this 🥲 
`<body bgcolor=1A4341 style=margin:135+185;color:F3AC3C;box-shadow:0+0+0+37q,0+70q,70q+0,66q+0,0+66q,0+0+0+69q#998235,0+0+0+5in;transform:rotate(225deg>`

**Reply by rafehimself** [reply]
that's way much more complicated for my brain like Mr G would say, I would need to play around with that feature much more to understand it,
how to know that path values ? also I don't get it how that apply if there is more than 1 element to be moved like in #84,
cuz in general that moves element from point a to b?
but thanks for sharing a neat trick 🙌 

hmm I wonder if there is better way to do those cuts in#82 those 4xsmall-box-shadows, (cuz 2 doesn't work) are not very elegant solution 😃

**Comment by padi** [comment]
@rafehimself

You're lucky you're playing in a Chromium-based browser because
I just spent an hour trying to figure out why I'm getting 88% in
Tesseract (https://cssbattle.dev/play/9) when both the
Slide & Compare and Diff show less-than-a-pixel
difference!

![Firefox Diff](https://user-images.githubusercontent.com/807110/157613980-b440844a-f6d8-4773-a080-7b1fc8d7a0b4.png)

I had a hunch so I opened my Chrome and loaded the same code.
It turns out, I was using `rotate` on Firefox, but it's not
working in Chrome... hence the 88% difference!!!

![Chrome Diff](https://user-images.githubusercontent.com/807110/157614278-ecf5e39d-b116-4a9f-b25a-60c2e6da1551.png)

My hunch is that when CSSBattle runs its test on the server,
it uses a headless chrome browser.

So I guess I'd have to stick to Chrome for now since playing
on Firefox may lead to false negatives 😅
